config/crds/resources/apiextensions.k8s.io_v1_customresourcedefinition_runjobs.run.cnrm.cloud.google.com.yaml

```diff
@@ -282,113 +268,121 @@ spec:
                                               - external
                                             properties:
                                               external:
-                                                description: 'Allowed value: The `version`
-                                                  field of a `SecretManagerSecretVersion`
-                                                  resource.'
+                                                description: ' If provided must be
```

Comment:
This exception is addressed in this PR to make it backward compatible. Even if we allow the `version` value, we don't want to encourage that, so I removed the comment. Open to feedback on whether we should put it back. @justinsb